runtime/compiler/runtime/ClassUnloadAssumption.cpp

```diff
@@ -1302,14 +1249,12 @@ J9::PersistentInfo::isUnloadedClass(
       auto clientData = TR::compInfoPT->getClientData();
       OMR::CriticalSection isUnloadedClass(clientData->getROMMapMonitor());
       return clientData->getUnloadedClassAddresses().mayContain((uintptr_t)v);
-      }
+      } 
```

Comment:
Please remove the blank at the end of this line.  It is causing the "Line Endings Check" error that is being reported on this file.